yt_dlp/postprocessor/embedthumbnail.py

```diff
@@ -91,8 +91,10 @@ def run(self, info):
         success = True
         if info['ext'] == 'mp3':
             # Method 1: Use mutagen
-            # Prioritize mutagen over ffmpeg since ffmpeg messes up the lyrics data
-            if mutagen:
+            if not mutagen:
+                self.to_screen('mutagen not was found. Falling back to ffmpeg. Lyrics may be corrupted')
```

Comment:
Not exactly corrupted. but formatting is changed which goes undetected by players(poweramp and musicolet).

Reply:
What would you reccomend for the message?

Reply:
wait, why are we warning about lyrics when embedding thumbnails?

Reply:
If lyrics is added and then thumbnail is embedded with ffmpeg, the lyrics becomes unreadable by players.

Though, on second thought, if mutagen don't exist, then lyrics could not have been added in the first place. So maybe we don't need the warning?

Reply:
I think this makes sense to keep just in case